setup.py

```diff
@@ -16,35 +16,39 @@
 readme = open('README.rst').read()
 history = open('CHANGES.rst').read()
 
-sphinx_require = 'Sphinx>=1.5.1'
+sphinx_require = 'Sphinx>=3'
 
 tests_require = [
     'pytest-mock>=1.6.0',
-    'pytest-invenio>=1.3.3,<2.0.0',
-    'invenio-accounts>=1.3.0,<2.0.0',
-    'invenio-app>=1.3.0,<2.0.0',
+    'pytest-invenio>=1.4.1',
+    'invenio-accounts>=1.4.3',
+    'invenio-app>=1.3.0',
+    'SQLAlchemy-Continuum>=1.3.6',
+    'psycopg2-binary>=2.8.6',
     sphinx_require,
 ]
 
 # Should follow inveniosoftware/invenio versions
-invenio_db_version = '>=1.0.5,<2.0.0'
-invenio_search_version = '>=1.4.0,<2.0.0'
+invenio_search_version = '>=1.4.0,<1.5.0'
```

Comment:
major: Are this lower bounds needed for tests to pass? By restricting more the pin (instead of leaving `<2.0.0` its basically breaking the dependency flow. This means, that in order to for example increase the version in `invenio-app-rdm` due to chained deps (invenio-rdm-records -> invenio-records-permissions) we will need to release this module too (and probably address things that were not noticed earlier by tests (i.e. the "release" tag of tests, which will tests compatibility with the latest versions.)

IMO If tests were failing we need to see why, if not I'd just leave the upper bound at 2.0.0